### PR TITLE
[1LP][RFR] fixture name change

### PR DESCRIPTION
**Pull request by sshveta**
Follow up PR for the changes in  https://github.com/ManageIQ/integration_tests/pull/6881

{{ pytest: -v --long-running --use-provider vsphere cfme/tests/ssui/test_ssui_dashboard.py cfme/tests/ssui/test_ssui_myservice.py cfme/tests/ssui/test_ssui_service_catalogs.py -k 'test_service_catalog_crud or test_myservice_crud or test_total_services' }}

### CODE REVIEW

**Comment by kedark3** [comment]
LGTM :+1: :fireworks:

**Comment by mkoura** [comment]
QUESTION: tests from test_ssui_myservice.py and test_ssui_service_catalogs.py are not collected for run by PRT, any reason?

File: cfme/fixtures/service_fixtures.py [hunk 1]
```diff
@@ -87,5 +87,5 @@ def order_service(appliance, provider, provisioning, vm_name, dialog, catalog, r
         partial_check=True)
     provision_request.wait_for_request(method='ui')
     assert provision_request.is_succeeded()
-    return catalog_item, provision_request
+    return catalog_item
 # TODO - remove request finally
```
Comment (jamesooden) [inline cfme/fixtures/service_fixtures.py]: Shouldn't this line be removed also?

Comment (sshveta) [reply cfme/fixtures/service_fixtures.py]: No , the request still has to be removed :)

**Comment by sshveta** [comment]
Running it for all tests but PRT is showing Internal error .

**Comment by mshriver** [comment]
@sshveta PRT 22786 is running with my modifications to PR description to collect all the modified tests.
It's failing within the order_service fixture:

```
    @pytest.fixture
    def order_service(appliance, provider, provisioning, vm_name, dialog, catalog, request):
        """ Orders service once the catalog item is created"""
    
        if hasattr(request, 'param'):
            param = request.param
            catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog,
                                               console_test=True if 'console_test' in param else None)
        else:
>           catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog)
E           TypeError: create_catalog_item() takes at least 6 arguments (5 given)
```

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ffbe3cf81a328d271991d9fc90ae539c9e235c4e:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 74:101: **E501** *line too long (103 > 100 characters)*
- :bangbang: Line 77:101: **E501** *line too long (103 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by sshveta** [comment]
Two failures not related to this PR .
They look related to the clean up vm change made recently . will investigate that separately .

File: cfme/fixtures/service_fixtures.py [hunk 2]
```diff
@@ -64,21 +65,28 @@ def create_catalog_item(appliance, provider, provisioning, vm_name, dialog, cata
...
                                            console_test=True if 'console_test' in param else None)
     else:
-        catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog)
+        catalog_item = create_catalog_item(appliance, provider, provisioning,
+                                           vm_name, dialog, catalog)
     service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
     service_catalogs.order()
     provision_request = appliance.collections.requests.instantiate(catalog_item.name,
-        partial_check=True)
+                                                                   partial_check=True)
```
Comment (kedark3) [inline cfme/fixtures/service_fixtures.py]: I think this line is over-indented. Please check, is it right?

Comment (sshveta) [reply cfme/fixtures/service_fixtures.py]: Yes. It's correct

Comment (jamesooden) [reply cfme/fixtures/service_fixtures.py]: Both are correct in [PEP 8's indentation rules|https://www.python.org/dev/peps/pep-0008/#indentation]

Comment (mshriver) [reply cfme/fixtures/service_fixtures.py]: We do prefer/enforce the hanging indent being lined up in that case though, so this change is correct.

**Pull request merged**